packages/app/src/components/Root/Root.tsx

```diff
@@ -68,6 +69,11 @@ export const Root = ({ children }: PropsWithChildren<{}>) => (
       <SidebarGroup label="Menu" icon={<MenuIcon />}>
         {/* Global nav, not org-specific */}
         <SidebarItem icon={HomeIcon} to="/" text="Home" />
+        <SidebarItem
+          icon={SchoolIcon}
+          to="learning-paths"
+          text="Learning Paths"
+        />
```

Comment:
Can we move this down between Docs and Clusters? The Catalog and APIs navigation are more important

Reply:
Resolved in the latest commit.